pkgs/os-specific/darwin/apple-source-releases/default.nix

```diff
@@ -254,6 +273,8 @@ let
 
     # TODO(matthewbauer):
     # To be removed, once I figure out how to build a newer Security version.
-    Security      = applePackage "Security/boot.nix" "osx-10.9.5"      "1nv0dczf67dhk17hscx52izgdcyacgyy12ag0jh6nl5hmfzsn8yy" {};
+    Security        = applePackage "Security/boot.nix" "osx-10.9.5"      "1nv0dczf67dhk17hscx52izgdcyacgyy12ag0jh6nl5hmfzsn8yy" {};
   };
+
+  packages = developerToolsPackages_11_3_1 // macosPackages_11_0_1 // stubPackages;
```

Comment:
Is the order here correct? Should stub packages take precedence over everything else?

I have #105026 locally and it tries to install `libutil-47.30.1-osx-10.12.6`, which doesn't seem right (and it also fails, because 10.12 doesn't know anything about aarch64).

cc @thefloweringash

cc

Reply:
> Is the order here correct? Should stub packages take precedence over everything else?

`stubPackages` does take precedence, the later overwrites the formers. 

> I have #105026 locally and it tries to install `libutil-47.30.1-osx-10.12.6`, which doesn't seem right (and it also fails, because 10.12 doesn't know anything about aarch64).

`libutil-47.30.1-osx-10.12.6` is in `stubPackages`, we haven't updated `libutil` yet, so it still installs the old 10.12 one, that is exactly expected. I'm a bit confused of which version is you actually want.